unit/Makefile

```diff
@@ -14,7 +15,9 @@ include ../src/common
 cprover.dir:
 	$(MAKE) $(MAKEARGS) -C ../src
 
-LIBS += ../src/ansi-c/ansi-c$(LIBEXT) \
+LIBS += ../src/java_bytecode/java_bytecode$(LIBEXT) \
+        ../src/miniz/miniz$(OBJEXT) \
+        ../src/ansi-c/ansi-c$(LIBEXT) \
```

Comment:
Sadly not possible - the order effects whether it compiles(!) - https://stackoverflow.com/questions/45135/why-does-the-order-in-which-libraries-are-linked-sometimes-cause-errors-in-gcc